docs/source/standard/contract.rst

```diff
@@ -94,3 +94,101 @@ Schema
 
     |ocdsDescription|
     All documents and attachments related to the contract, including any notices.
+
+.. _ContractTerms:
+
+ContractTerms
+=============
+
+Schema
+------
+
+:contractType:
+    string, required
+
+    The only value for this field is ``lease``.
+
+:leaseTerms:
+    :ref:`LeaseTerms`
+
+    The options of property lease.
+
+.. _LeaseTerms:
+
+LeaseTerms
+==========
+
+Schema
+------
+
+:leaseDuration:
+    `ISO8601 duration`_ object, required
+
+    Property lease duration in contract.
+
+:taxHolidays:
+    list of :ref:`TaxHolidays`, optional
+
+    A period during which a person or company is allowed to pay no payment or less than usual.
+
+:escalationClauses:
+    list of :ref:`EscalationClauses`
+
+.. _TaxHolidays:
+
+TaxHolidays
+===========
+
+Schema
+------
+
+:taxHolidaysDuration:
+    `ISO8601 duration`_ object, required
+
+    Duration of lease holidays.
```

Comment:
Duration of the period within which a person or company is allowed to pay less than usual (or provide no payment).